CONTRIBUTING.md

```diff
@@ -4,4 +4,4 @@ Thank you for your interest in contributing to Taichi! Please check out the [Con
 
 ## Developer Installation
 
-Please follow the [Developer Installation](https://docs.taichi.graphics/lang/articles/contribution/dev_install) doc to learn how to build Taichi from scratch.
+Taichi is mainly developed in C++17 and Python3. Please check out the [Developer Installation](https://docs.taichi.graphics/lang/articles/contribution/dev_install) to build Taichi from scratch. Note that Taichi currently depends on LLVM-10.0.0, and we recommned to use [our prebuilt LLVM libraries](https://docs.taichi.graphics/lang/articles/contribution/dev_install#installing-dependencies) for each platform.
```

Comment:
Taichi is developed mainly in C++17 and Python3. Please check out the [Developer Installation](https://docs.taichi.graphics/lang/articles/contribution/dev_install) to build Taichi from scratch. Note that Taichi is LLVM-10.0.0 dependent and that we recommend installing [our pre-built LLVM libraries](https://docs.taichi.graphics/lang/articles/contribution/dev_install#installing-dependencies) for your platform.

Reply:
Shall we be more explicit by saying "build Taichi from source"? IMO, this is more commonplace?

Reply:
```suggestion
Taichi is developed mainly in C++17 and Python3. Please check out the [Developer Installation](https://docs.taichi.graphics/lang/articles/contribution/dev_install) to build Taichi from scratch. Note that Taichi is LLVM-10.0.0 dependent and that we recommend installing [our pre-built LLVM libraries](https://docs.taichi.graphics/lang/articles/contribution/dev_install#installing-dependencies) for your platform. 
```